quickstart-webauthn.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  kratos-migrate:
+    volumes:
+      - type: bind
+        source: ./contrib/quickstart/kratos/webauthn
+        target: /etc/config/kratos
+
+  kratos:
+    volumes:
+      - type: bind
+        source: ./contrib/quickstart/kratos/webauthn
+        target: /etc/config/kratos
+
+  kratos-selfservice-ui-node:
+    environment:
+      - KRATOS_BROWSER_URL=http://localhost:4433/
```

Comment:
This overrides the variable from the standalone quickstart with `localhost` instead of the local IP. WebAuthN only works on domains, not IPs so this is important to get this to work.